src/api/public/apidocs-new/paths/person_login_token.yaml

```diff
@@ -40,6 +40,87 @@ get:
           example:
             code: view_user_not_authorized
             summary: 'You are not authorized to view this User.'
+  tags:
+    - Person
+
+post:
+  summary: Create a new authentication token for a person.
+  description: |
+    Create a new authentication token for a person passed as parameter.
```

Comment:
```suggestion
    Create a new authentication token for a person.
```

Reply:
the passed as parameter part sound a bit strange to me, I think its not needed :)

Reply:
Yes. Better without that part.

I addressed this and the rest of your suggestions. Please have another look @krauselukas